tests/queries/0_stateless/02151_client_option_echo.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# single query echo on
+${CLICKHOUSE_CLIENT} --echo --query="DROP TABLE IF EXISTS echo_test_0" && \
+# single query echo off
+${CLICKHOUSE_CLIENT} --query="DROP TABLE IF EXISTS echo_test_1" && \
+# multi query echo on
+${CLICKHOUSE_CLIENT} --echo --multiquery --query="DROP TABLE IF EXISTS echo_test_2;DROP TABLE IF EXISTS echo_test_3" && \
+# multi query echo off
+${CLICKHOUSE_CLIENT} --multiquery --query="DROP TABLE IF EXISTS echo_test_4;DROP TABLE IF EXISTS echo_test_5" && \
+
+true
```

Comment:
What does this line do?

Reply:
`true` is supposed to be ending the chain of commands connected with `&&`. I think it is a common practice when writing chained commands so all of them will end with `&& \` and can be easily rearranged. The stray blanc line probably got there by accident. Is command chaining actually required for these tests? 

Also i see that Fast test (actions) failed with message `line 4: --echo: command not found`. Perhaps `CLICKHOUSE_CLIENT` did not expand properly? Or perhaps this test must be registered somewhere to use correct environment?

Reply:
Oh, I missed that commands are chained. So no, there is no need to chain them, the errors will be written to stderr and if something goes wrong the test will fail anyway :)

I suggested a change and your test should pass with it

Reply:
Oh, I missed that commands are chained. So no, there is no need to chain them, the errors will be written to stderr and if something goes wrong the test will fail anyway :)

I suggested a change and your test should pass with it

Reply:
Alright, i've applied these changes and removed chaining.